OpenRA.Mods.Common/Scripting/Properties/InstantlyRepairsProperties.cs

```diff
@@ -0,0 +1,41 @@
+#region Copyright & License Information
+/*
+ * Copyright (c) The OpenRA Developers and Contributors
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Linq;
+using OpenRA.Mods.Common.Activities;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Scripting;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Scripting
+{
+	[ScriptPropertyGroup("Ability")]
+	public class InstantlyRepairsProperties : ScriptActorProperties, Requires<IMoveInfo>, Requires<InstantlyRepairsInfo>
+	{
+		readonly InstantlyRepairs[] instantlyRepairs;
+
+		public InstantlyRepairsProperties(ScriptContext context, Actor self)
+			: base(context, self)
+		{
+			instantlyRepairs = Self.TraitsImplementing<InstantlyRepairs>().ToArray();
+		}
+
+		[ScriptActorPropertyActivity]
+		[Desc("Repair the target actor instantly.")]
+		public void InstantlyRepairs(Actor target)
+		{
+			// NB: Scripted actions get no visible targetlines.
```

Comment:
You need to add the target line color as a parameter to the existing constructor, otherwise it will use the color that is defined in the info.

Reply:
I think I will remove this comment.

Reply:
comment removed

Reply:
No, the comment is correct. It should not show target lines.

Reply:
Add the comment back and fix the target line.